ajax/libs/gijgo/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "gijgo",
+  "version": "1.3.0",
+  "description": "Gijgo is a set of free open source javascript controls distributed under MIT License. All widgets are high performance, built on top of the jQuery JavaScript Library with built-in support for Bootstrap, Material Design and Font Awesome. They are designed to saves you time and scales with your development process.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/atatanasov/gijgo.git"
+  },
+  "homepage": "http://gijgo.com/",
+  "filename": "js/gijgo.min.js",
+  "keywords": [
+    "jquery-plugin",
+    "jquery",
+    "gijgo",
+    "javascript",
+    "plugin",
+    "grid",
+    "datagrid",
+    "table",
+    "dialog",
+    "modal",
+    "draggable",
+    "droppable",
+    "tree",
+    "treeview"
+  ],
+  "author": {
+    "name": "Atanas Atanasov",
+    "email": "nascovt@gmail.com",
+    "url": "http://gijgo.com"
+  },
+  "license": "MIT",
+  "npmName": "gijgo",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```

Comment:
Let's change filemap:
from
```js
"**/*"
```
to
```js
"js/**/*.+(js|map|css)",
"css/*"
```
Because we don't need to host `README.md`.
Note:
1. `basePath` is the path in the npmjs tarball; it will be ignored when files are copied to cdnjs
2. `files` indicates the file(s) to copy
ref: https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md#2-configure-auto-update